grammar.ebnf

```diff
@@ -20,8 +20,19 @@ StructDecl = "struct", Identifier, "{", { StructField }, "}";
 StructField = Identifier, ":", Type, ";";
 
 
+Statement = "{", { Statement }, "}"
+          | "while", "(", Expression, ")", Statement
+          | Identifier, "=", Expression, ";"
+          | Identifier, "[", Expression, "]", "=", Expression, ";"
+          | "if", "(", Expression, ")", Statement
+          | Identifier, "(", FuncCallArgList, ")", ";" 
```

Comment:
For educational purposes in case you don't have it in your mind, in languages like C / C++, the `Identifier` here should be an expression, because in `p->a(1, 2)`, `a` might be a function pointer. And a function pointer can result from arbitrary expressions. In C++ it can result from method referencing etc.
Here we don't have function pointers or methods (i.e. member functions) so no need for that.

Reply:
Yes, that is why I did not make it an expression in the first place. Thanks for the note nonetheless.